jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -95,10 +95,10 @@ public CRIUSupport(Path imageDir) {
 	/**
 	 * Queries if CRIU support is enabled and criu library has been loaded.
 	 *
-	 * @return TRUE if support is enabled and the library is loaded, FALSE otherwise
+	 * @return TRUE if CRIU support is enabled and the library is loaded, FALSE otherwise
 	 */
 	public static boolean isCRIUSupportEnabled() {
-		return InternalCRIUSupport.isCRIUSupportEnabledAndNativeLoaded();
+		return InternalCRIUSupport.isCRIUSupportEnabled() && InternalCRIUSupport.isNativeLoaded();
```

Comment:
I meant, this should be `isCRIUSupportEnabledAndNativeLoaded`

Reply:
Sure, keep the existing `InternalCRIUSupport::isCRIUSupportEnabledAndNativeLoaded()`, and the new `isNativeLoaded()`.